source/extensions/omni.isaac.lab/omni/isaac/lab/sim/spawners/sensors/sensors_cfg.py

```diff
@@ -129,7 +130,8 @@ def from_intrinsic_matrix(
             width: Width of the image (in pixels).
             height: Height of the image (in pixels).
             clipping_range: Near and far clipping distances (in m). Defaults to (0.01, 1e6).
-            focal_length: Perspective focal length (in cm). Defaults to 24.0 cm.
+            focal_length: Perspective focal length (in cm) used to calculate pixel size. Defaults to None. If None
+                focal_length will be calculated 1 / width.
```

Comment:
@pascal-roth Sorry that was a mistype. pixel_size will be 1/width instead of calculated from focal_length. We actually shouldn't need focal length because the intrinsic matrix has all the info we need. I left focal length in there so it doesn't break backwards compatibility.
```suggestion
                pixel_size will be calculated 1 / width.
```